README.md

```diff
@@ -1 +1,12 @@
 # link-tool-app
+
+## Dev setup
+
+### Requirements
+
+* npm 6
+* node 14
```

Comment:
these are the versions that I happen to have, they might not match yours but they definitely work